python/mujincontrollerclient/uriutils.py

```diff
@@ -598,7 +603,7 @@ def partType(self):
         else:
             basePartType = _Unquote(self._primaryKey)
         if self._fragment:
-            return basePartType + self._fragmentSeparator + _EnsureUTF8(self._fragment)
+            return basePartType + self._fragmentSeparator + self._fragment
```

Comment:
need to unquote fragment?

Reply:
@Hsifnus

Reply:
partType is unicode, so I think this is fine.

Reply:
`_fragment` is already unicode and unquoted